.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,38 @@
+name: I think I found a bug in SQLx
+description: Create a bug-report issue
+labels: [bug]
+body:
+  - type: textarea
+    id: related-issues
+    validations:
+      required: true
+    attributes:
+      label: I have found these related issues/pull requests
+      description: "I have searched by clicking [HERE](https://github.com/launchbadge/sqlx/issues?q=) for existing issues, these are the ones I've found, and this is why I think this deserves a new issue."
```

This is better formatted as a multi-line string: https://yaml-multiline.info/

```suggestion
      description: |
        I have searched by clicking [HERE](https://github.com/launchbadge/sqlx/issues?q=) 
        for existing issues, these are the ones I've found, 
        and this is why I think this deserves a new issue.
````